### PR TITLE
Fixed pause

### PR DESCRIPTION
**Pull request by DavertMik**
## Motivation/Description of the PR

Applicable helpers:

- [ ] Webdriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe

- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [ ] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- lib/pause.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4871039&bid=16149034)

**Comment by DavertMik** [comment]
@jdgreenberger  please check, I updated the previous PR, now objects are passed as is without stringifying

File: lib/pause.js [hunk 1]
```diff
@@ -14,11 +14,12 @@ let rl;
 let nextStep;
 let finish;
 let next;
-let registeredVariables;
+let registeredVariables = {};
 /**
  * Pauses test execution and starts interactive shell
  */
-const pause = function (passedObject) {
+const pause = function (passedObject = {}) {
```
Comment (DavertMik) [inline lib/pause.js]: Codacy Issue found: [Block must not be padded by blank lines.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4871039)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- lib/pause.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4871039&bid=16149034)

File: lib/pause.js [hunk 2]
```diff
@@ -85,6 +91,10 @@ function parseInput(cmd) {
 
     const result = executeCommand instanceof Promise ? executeCommand : Promise.resolve(executeCommand);
     result.then((val) => {
+      if (isCustomCommand) {
+        console.log(val);
```
Comment (PeterNgTr) [inline lib/pause.js]: could we use `output.print` here?

Comment (DavertMik) [reply lib/pause.js]: Not really, this can be a very arbitrary output so I'd like to fully delegate that to NodeJS without any customization

**Pull request merged**